### DISCUSSION TITLE
Docs download for offline use

### DISCUSSION THREAD

**Discussion by VALERYRAMIREZ** [fastapi/fastapi#12185] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No code
```

### Description

Hi all, how are you?. I'm looking for a way to download de docs for offline use.

How can I do it?

Thanks in advance for the help

### Operating System

Linux

### FastAPI Version

0.113.0

### Pydantic Version

2.9.0

### Python Version

3.10.6

**Comment by JavierSanchezCastro** [comment, marked as answer]
https://fastapi.tiangolo.com/contributing/#docs
